### PR TITLE
moved distributed tracing terminology

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
CC: @open-telemetry/specs-approvers

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "rewrap.wrappingColumn": [80,120],
```
Comment (bogdandrutu) [inline .vscode/settings.json]: Why this file? Consider to move this in a separate PR if needed and explain how to use this.

File: terminology.md [hunk 2]
```diff
@@ -1 +1,112 @@
...
+- An operation name
+- A start and finish timestamp
+- A set of zero or more key:value **Attributes**. The keys must be strings. The
+  values may be strings, bools, or numeric types.
+- A set of zero or more **Events**, each of which is itself a key:value map
+  paired with a timestamp. The keys must be strings, though the values may be of
+  the same types as Span **Attributes**.
+- [**Links**](#links-between-spans) to zero or more causally-related **Spans**
+  (via the **SpanContext** of those related **Spans**).
+- **SpanContext** identification of a Span. See below.
```
Comment (bogdandrutu) [inline terminology.md]: Probably a "parentSpanId" to show that an explicit parent is defined here but the trace can have links as well?
